src/test/java/com/yelp/nrtsearch/server/warming/WarmerTest.java

```diff
@@ -205,33 +206,41 @@ private List<SearchRequest> getTestSearchRequests() {
 
   private List<SearchRequest> getTestBasicSearchRequests() {
     List<SearchRequest> testRequests = new ArrayList<>();
-    SearchRequest searchRequest =
-        SearchRequest.newBuilder()
-            .setIndexName(index)
-            .setQuery(Query.newBuilder().setTermQuery(TermQuery.newBuilder().setField("field0")))
-            .build();
-    testRequests.add(searchRequest);
-
-    searchRequest =
-        SearchRequest.newBuilder()
-            .setIndexName(index)
-            .setQuery(
-                Query.newBuilder()
-                    .setFunctionScoreQuery(
-                        FunctionScoreQuery.newBuilder()
-                            .setQuery(
-                                Query.newBuilder()
-                                    .setTermQuery(TermQuery.newBuilder().setField("field1")))
-                            .setScript(Script.newBuilder().setLang("js").setSource("3 * 5"))))
-            .build();
-    testRequests.add(searchRequest);
+    int i = 0;
+    for (; i < 2; i++) {
+      SearchRequest searchRequest =
+          SearchRequest.newBuilder()
+              .setIndexName(index)
+              .setQuery(
+                  Query.newBuilder().setTermQuery(TermQuery.newBuilder().setField("field" + i)))
+              .build();
+      testRequests.add(searchRequest);
+    }
+
+    for (; i < 4; i++) {
```

Comment:
minor suggestion: use `for (int i = 2; i < 4; i++)` to make it clear that this for loop will add 2 requests. When I first read it quickly, I thought it was adding 4.